src/main/java/ui/AvatarUi.java

```diff
@@ -0,0 +1,57 @@
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░▓███▓▓▓▓████▒▒▒▓████▓▓▓▓██▓▒░░░░░░░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░▓██▓▒▓█▓▓▓▓▒▓▒▒▓▒▓▓▓▓▓▒▓██▓▒░░░░░░░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░▓▓█▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▓██▓▒░░░░░░░░░░░░░░░░░░░▒░░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░▓▓█▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▓█▓▒░░░░░░░░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░▒▓▓▓▒▒▒▒▒▒▓▒▒▓▒▒▒▒▒▓▓▓▓▒░░░░░░░░░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▒▒▒▒▒▓▓▓▓▒▒▒▒▒▓▓▓░░░░░░░░░░░░░░░░░░░░░░░▒░░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▒▒▒▒▓▒▒▓▒▒▒▒▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░▒░░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░▒▓▓▓▒▒▒▓▓▓▓▓▓▒▒▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░▒░░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░██▓▓▒▒▒▒▒▒▒▒▒▒▓▓█▓░░░░░░░░░░░░░░░░░░░░░░░░▒░░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░▒█▓▓█▓▓▒▒▒▒▒▒▓▓█▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓█▓▓▓▓▓██▓▒▒▓▒▓▒░░░░░░░░░░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "░░▒▒░░░░░░░▒░░░░░░░░░░░░░░░▒██▒▒▒▒▓▓▓▓▓▓▓▓▓▓▒▒▒▒▒▓██▒░░░░░░░░░░░░░░░░░░░░▒▒░░▒▒▒\n" +
+            "░░▒▒░░░░▒▒▓▒▒▒▒▒░░░░░░░░▒▓████▒▒▒░░░▒▓▓▓▓▒▒▒░░░▒▒▒████▓▒░░░░░░░░░░░░░░░░░▒▒░░▒▒▒\n" +
+            "░░▒▒░░▒▒▓▓▓▓▓▒░░░░░░▒▓▓███████▒▒░░░░░░▒▓▓▒░░░░░▒▒▒███████▓▓▒░░░░░░░░░░░░░▒▒░░▒▒░\n" +
+            "▒▒▒▒░░░▒▓▓▓▓▒▒░▒▓▓██████████▓█▒▒░░░░▒██████░░░░░▒▒█▓██████████▓▓▓▒▒▒▒▒▒▒▒▒▒░░▒▒░\n" +
+            "▓▓▓▓▒▒▒▒▓▓████████████████▓▓▓█▒▒░░░▒▒██████▒▒░░░▒▒█▓▓▓███████████████▓▓▓▓▓▒▒▒▒▒▒\n" +
+            "▓▓▓▓▒▒▓██████████▓▓▓▓▓██▓▓▓▓▓█░░░░▒░░▒████░░░▒░░░▒█▓▓▓▓▓██▓▓▓▓▓▓███████████▓▓▓▒▒\n" +
+            "▓▓▓▓█████████▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓█▒░░░░░░░████░░░░░░░▓█▓▓▓▓▓▓▓▓▓▓█▓▓▓▓▓▓████████▓▓▓▓\n" +
+            "███▓███▓█▓▓▓▓▓▓▓▓▓▓█▓▓█▓▓▓▓▓▓█▓░░░░░░▓████▒░░░░░░██▓▓▓▓▓██▓▓█▓▓▓▓▓▓▓▓▓▓▓████▒▒▒▒\n" +
+            "▒▒▒▒███▓█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓██░░░░░░█████▓░░░░░▒██▓▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓▓█▓███▓▓▓▓\n" +
+            "▓▓▓▓████▓█▓▓▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓██▓░░░░░██████░░░░░▓██▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓▓▓██████▓██\n" +
+            "████████▓██▓▓▓▓▓▓▓▓▓██▓▓▓▓▓▓▓██▓▒░░░▒██████▒░░░▒▓██▓▓▓▓▓▓▓▓█▓▓▓▓▓▓▓▓▓██▓█████▓▓▓\n" +
+            "▓▓▓████████████▓▓▓▓▓▓██▓▓▓▓▓▓██▓▓░░░▓██████▒░░░▓▓██▓▓▓▓▓▓███▓▓▓▓▓▓▓███████████▓▓\n" +
+            "░░░█████████████████▓██████████▓▓▓░░▓██████▓░░▓▓▓███████████████████████████████\n" +
+            "▒▒▓█████████████████████████████▓▓▒░▓███████░▒█▓▓█████████████████████████████▓▓\n" +
+            "█▓█████████████████████████████████▒▓███████░▓▓▓████████████▓▓▓▓▓█████████████▒▒\n" +
+            "█▒██████████████████████████████████████████▓████████████▓▓▓▓█████████████████▓▒\n" +
+            "█▒████████████████████████████████████████████████████████████████████████████▓▒\n" +
+            "▓▓█████████████████████████████████████████████████████████████████████████████▒\n" +
+            "▓▓█████████████████████████████████████████████████████████████████████████████▒\n";
```

Comment:
Thanks for adding in such a detailed an professional looking ASCII avatar!

Reply:
Looks amazing!